### PR TITLE
`Assessment`: Fix responsive layout for exercise assessment dashboard

### LINKED ISSUE #5362: Assessment Dashboard responsiveness
### Describe the bug

It seems that something is wrong with the responsiveness.
I noticed that this problem is already present in 5.8.7 in production.

### To Reproduce

1. Go to `course-management/xxx/assessment-dashboard/xxx`
2. Resize the window
3. See overlapping elements

### Expected behavior

I would expect a good responsive design.

### Screenshots

image

https://user-images.githubusercontent.com/60005702/178160435-a7d20b03-0d9e-4954-a49d-3a33a077d5de.mp4

### What browsers are you seeing the problem on?

Firefox

### PR DESCRIPTION
**Pull request by ge65cer**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] Following the [theming guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context

The mobile layout of the exercise assessment dashboard could use some improvements.

### Description

The layout no longer overlaps and should now shrink accordingly. Fixes #5362.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Exercise

1. Log in to Artemis
2. Navigate to Course Administration
3. Go the _Exercise_ Assessment Dashboard of the exercise you created
4. Ensure the layout does not overlap and shrinks correctly
5. Ensure that the responsive layout for the _Course_ Assessment Dashboard remains unchanged

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### CODE REVIEW

**Review by luisa-we — COMMENTED** [review]
Tested on TS2. For me, the appearance was different from your screenshots.

When I didn't have the Exercise Assessment Dashboard on full screen on my laptop, there was a lot of white space on the right side:
Bildschirmfoto 2022-07-11 um 14 25 50

And on my phone, the stats overlapped so it was hard to read some of them:

**Review by luisa-we — DISMISSED** [review]

**Review by jbllhff — DISMISSED** [review]
Tested on ts2 with chrome dev tools, looks good!

File: src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html [hunk 1]
```diff
@@ -5,71 +5,69 @@
...
-                                [togglingSecondCorrectionButton]="togglingSecondCorrectionButton"
-                            ></jhi-second-correction-enable-button>
-                            <a class="btn btn-primary btn-sm ms-1" [routerLink]="getSubmissionsLinkForExercise(exercise)">
-                                <span jhiTranslate="artemisApp.assessmentDashboard.submissions">Submissions</span>
-                            </a>
-                        </ng-container>
-                    </div>
-                </div>
+        <div class="col-12 col-md-7 col-xl-4">
+            <p class="h3" style="text-align: left">
```
Comment (Dominik-Weinzierl) [inline src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html:9]: There is a scss file - i suggest to move it :+1: (or: Can you use bootstraps classes?)

File: src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html [hunk 2]
```diff
@@ -5,71 +5,69 @@
...
+                [exercise]="exercise"
+                [tutorParticipation]="tutorParticipation"
+                [numberOfSubmissions]="numberOfSubmissions"
+                [totalNumberOfAssessments]="totalNumberOfAssessments"
+                [numberOfComplaints]="complaintsDashboardInfo.total"
+                [numberOfOpenComplaints]="complaintsDashboardInfo.total - complaintsDashboardInfo.done!"
+                [numberOfMoreFeedbackRequests]="moreFeedbackRequestsDashboardInfo.total"
+                [numberOfOpenMoreFeedbackRequests]="moreFeedbackRequestsDashboardInfo.total - moreFeedbackRequestsDashboardInfo.done!"
+                [numberOfAssessmentsOfCorrectionRounds]="numberOfAssessmentsOfCorrectionRounds"
+                style="display: block"
```
Comment (Dominik-Weinzierl) [inline src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html:23]: same

File: src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html [hunk 3]
```diff
@@ -5,71 +5,69 @@
...
+            >
+            </jhi-tutor-participation-graph>
+            <p style="text-align: left; float: left; font-size: 16px">
```
Comment (Dominik-Weinzierl) [inline src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html:26]: same

File: src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html [hunk 4]
```diff
@@ -5,71 +5,69 @@
...
-                        [animations]="false"
-                        [trimLabels]="false"
-                        (select)="navigateToExerciseSubmissionOverview($event)"
-                    >
-                        <ng-template #tooltipTemplate let-model="model"> {{ ((model.value * 100) / this.numberOfSubmissions.total).toFixed(2) }}% </ng-template>
-                    </ngx-charts-pie-chart>
-                </div>
+            <div
+                *ngIf="isAtLeastInstructor && exam?.numberOfCorrectionRoundsInExam && exam!.numberOfCorrectionRoundsInExam! > 1 && isExamMode && !isTestRun"
+                style="display: inline-block"
```
Comment (Dominik-Weinzierl) [inline src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html:32]: same

**Review by Dominik-Weinzierl — COMMENTED** [review]
I think we should also do some refactoring (e.g avoid. style in html file). I know its not your code - so sorry :)

**Review by jschmautzer — DISMISSED** [review]
tested on ts2, worked as described

**Comment by joaquinmurcia** [comment]
Tested on TS1. Works as expected.

**Review by alaraozdenler — DISMISSED** [review]
Tested on ts2, works as expected

**Review by Mikael31 — APPROVED** [review]
tested on ts3. worked fine

**Review by bassner — APPROVED** [review]
Tested on TS5, works good. Code LGTM 👍

**Review by SimonPaul02 — APPROVED** [review]
Tested on TS1, works as expected

**Review by yuling-sun — APPROVED** [review]
Tested on TS1. Works well.

**Review by maximiliansoelch — APPROVED** [review]
Tested on TS1, works as expected. Code looks good 👍

**Pull request merged**